src/js/AddFormUI.js

```diff
@@ -40,16 +42,19 @@ class AddFormUI {
       this.addBtn.style.display = 'block';
     });
```

Comment:
```suggestion
    this.addBtn.style.display = 'block';
    this.input.value = "";
    });
```

I think when you cancel the addition, the input should `""`

Reply:
yep, it makes sense